internal/ui/pages/active-search/index.templ

```diff
@@ -0,0 +1,69 @@
+package activesearch
+
+import "strings"
+
+type Row struct {
+	FirstName string
+	LastName  string
+	Email     string
+}
+
+css classLayout() {
+	display: grid;
+}
+
+templ layout(header, body templ.Component) {
+	<main class={ classLayout() }>
+		<div>
+			@header
+		</div>
+		<div>
+			@body
+		</div>
+	</main>
+}
+
+templ body() {
+	<h2>Active Search:</h2>
+	<h3>
+		Search Contacts
+		<span class="htmx-indicator">
+			<img src="/img/bars.svg"/> Searching...
+		</span>
+	</h3>
+	<input
+		type="search"
+		name="search"
+		placeholder="Begin Typing To Search Users..."
+		hx-post="/active-search/search"
+		hx-trigger="input changed delay:500ms, keyup[key=='Enter'], load"
+		hx-target="#search-results"
+		hx-indicator=".htmx-indicator"
+	/>
+	<table class="table">
+		<thead>
+			<tr>
+				<th>First Name</th>
+				<th>Last Name</th>
+				<th>Email</th>
+			</tr>
+		</thead>
+		<tbody id="search-results"></tbody>
+	</table>
+}
+
+templ searchResults(rows []Row, searchQuery string) {
+	for _, v := range rows {
```

Comment:
the logic should be in the handler file, the templ file is just for layout, pass a pre filtered rows into this instead of doing the search logic here